<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paradiddle Companion</string>
  </property>
  <widget class="QWidget" name="songConversion">
   <widget class="QProgressBar" name="conversionProgress">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="convertAll">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>410</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convertSelected">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>410</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert Select</string>
    </property>
   </widget>
   <widget class="QListWidget" name="conversionList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>181</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="conversionTabs">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>491</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="midiTab">
     <attribute name="title">
      <string>MIDI</string>
     </attribute>
     <widget class="QToolButton" name="midiFileButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="midiFileTextBox">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="midiFileLabel">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>66</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>MIDI File:</string>
      </property>
     </widget>
     <widget class="QLabel" name="midiTrackLabel">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>81</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>MIDI Track:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="midiTrackComboBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="advancedLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>131</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Advanced Settings:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="accentCheckBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Accent Notes</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="ghostCheckBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>250</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ghost Notes</string>
      </property>
     </widget>
     <widget class="Line" name="midiToAdvancedSplit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>461</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="accentVelSpinBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>210</y>
        <width>45</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximum">
       <number>127</number>
      </property>
      <property name="value">
       <number>110</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="ghostVelSpinBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>250</y>
        <width>45</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximum">
       <number>127</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
     </widget>
     <widget class="QLabel" name="accentVelLabel">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>210</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Velocity Threshhold:</string>
      </property>
     </widget>
     <widget class="QLabel" name="ghostVelLabel">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>250</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Velocity Threshhold:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="audioTab">
     <attribute name="title">
      <string>Audio</string>
     </attribute>
     <widget class="QLabel" name="drum1Label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>60</y>
        <width>101</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Drum Track 1:</string>
      </property>
     </widget>
     <widget class="QLabel" name="song1Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Song Track 1:</string>
      </property>
     </widget>
     <widget class="QLabel" name="song2Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Song Track 2:</string>
      </property>
     </widget>
     <widget class="QLabel" name="song3Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Song Track 3:</string>
      </property>
     </widget>
     <widget class="QLabel" name="song4Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Song Track 4:</string>
      </property>
     </widget>
     <widget class="QLabel" name="song5Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Song Track 5:</string>
      </property>
     </widget>
     <widget class="QLabel" name="drum2Label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>100</y>
        <width>101</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Drum Track 2:</string>
      </property>
     </widget>
     <widget class="QLabel" name="drum3Label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>140</y>
        <width>101</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Drum Track 3:</string>
      </property>
     </widget>
     <widget class="QLabel" name="drum4Label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>180</y>
        <width>101</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Drum Track 4:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="song1TextBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="song2TextBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="song3TextBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="song4TextBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>160</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="song5TextBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>200</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="drum1TextBox">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>60</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="drum2TextBox">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>100</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="drum4TextBox">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>180</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="drum3TextBox">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>140</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="song1Button">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="song5Button">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>200</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="song4Button">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>160</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="song3Button">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>120</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="song2Button">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="drum1Button">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>60</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="drum2Button">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>100</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="drum3Button">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>140</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="drum4Button">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>180</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="yamlLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>71</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>YAML File:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="drumsetTextBox">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>280</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="yamlTextBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>280</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="yamlButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>280</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="drumsetLabel">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>280</y>
        <width>71</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Drumset:</string>
      </property>
     </widget>
     <widget class="QToolButton" name="drumsetButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>280</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="Line" name="trackToFileLineSplit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>461</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="metadataTab">
     <attribute name="title">
      <string>Metadata</string>
     </attribute>
     <widget class="QLabel" name="difficultyLabel">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>130</y>
        <width>66</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Difficulty:</string>
      </property>
     </widget>
     <widget class="QLabel" name="albumNameLabel">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>70</y>
        <width>101</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Album Name: </string>
      </property>
     </widget>
     <widget class="QLabel" name="songNameLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Song Name:</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="artistNameTextBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QLabel" name="artistNameLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Artist Name:</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="albumNameTextBox">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>90</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="charterNameTextBox">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="coverImageButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>150</y>
        <width>26</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="coverImageTextBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QTextEdit" name="descriptionTextBox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>259</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="difficultyComboBox">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>150</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="descriptionLabel">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>230</y>
        <width>101</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Description:</string>
      </property>
     </widget>
     <widget class="QLabel" name="converImageLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>141</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Cover Image Path:</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="songNameTextBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QLabel" name="charterNameLabel">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>101</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Charter Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="outputLabel">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>190</y>
        <width>101</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Output Path:</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="outputTextBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>190</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="outputButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>190</y>
        <width>26</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>668</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="openChartAction"/>
    <addaction name="openFolderAction"/>
    <addaction name="separator"/>
    <addaction name="actionImport_Chart"/>
    <addaction name="actionImport_Folder_of_Charts"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="midiCompanionAction"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="openChartAction">
   <property name="text">
    <string>Open Chart</string>
   </property>
  </action>
  <action name="openFolderAction">
   <property name="text">
    <string>Open Folder of Charts</string>
   </property>
  </action>
  <action name="midiCompanionAction">
   <property name="text">
    <string>MIDI Companion</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionImport_Chart">
   <property name="text">
    <string>Import Chart</string>
   </property>
  </action>
  <action name="actionImport_Folder_of_Charts">
   <property name="text">
    <string>Import Folder of Charts</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
